packages/light-client/package.json

```diff
@@ -82,6 +82,14 @@
     "qs": "^6.11.1",
     "uint8arrays": "^5.0.1"
   },
+  "peerDependencies": {
```

Comment:
do we have to declare this here again, or will this be inferred from the bls package already?

Reply:
Pretty sure that it will show up as a peerDep warning in the command line output but figured it was worth explicitly listing so consumers will know to dig deeper and not just ignore the install output

Reply:
I removed. just waiting for CI @nflaig

Reply:
> Pretty sure that it will show up as a peerDep warning in the command line 

Since it's set as optional peer dep it might not be shown at all, depends on the package manager though, for example, npm installs peer dependencies that are not optional automatically if the engine is supported by the package.

> but figured it was worth explicitly listing so consumers will know to dig deeper and not just ignore the install output

that's good point, but I think this is something we need to document in the README of the light client to give it more visiblity

Reply:
I removed it and we added a flag to allow for swapping implementations to the light-client.  I added a note to the docs about it.